third-party/realdds/readme.md

```diff
@@ -1,39 +1,66 @@
+
+Multiple *participants* can be part of this system: multiple clients can access a single server. I.e., the server broadcasts data rather than hand it to one particular client. Likewise, multiple servers can exist together (if the network bandwidth permits).
+
+### Discovery
+
+To bring clients and servers together, a [discovery](doc/discovery.md) mechanism at the device level (called the *device-info*) is provided in the form of a topic with a fixed name. A server would broadcast on this topic, while a client would watch it.
+
+### Server
+
+Any entity that provides a data stream on a specific topic is taking on the role of a *stream-server*.
+
+Any entity that puts one or more stream-servers together for publication, with attached mechanisms for control & notification, is a *device-server*.
+
+### Client
+
+Consumes data produced by a server: a *stream* is the client to a stream-server, and a *device* to a device-server.
+
+#### ROS2
+
+[ROS2](https://docs.ros.org/) is one of the clients for which effort has been expended in order to be compliant. This compliance is direct-to-ROS compliance that does not require use of a custom node (based on librealsense) for translation (as the [ROS wrapper for librealsense](https://github.com/IntelRealSense/realsense-ros) is): ROS is a DDS participant like any other Client.
+
+Specifically, at this time, the requirement is that ROS nodes should be able to:
+1. See any streams that RealDDS exposes
+2. Understand the (default) formats used by those streams
+
+As RealDDS is improved, more and more functionality will work directly with ROS.
 
-This stand-alone library aims to help control all Intel RealSense camera functionality.
+# Terminology
 
-The objects defined can help to create two types of applications camera's server and user.
+Most objects in RealDDS are prefixed with `dds_` and exist in the `realdds::` namespace. When a `device-server` is mentioned, the corresponding object is likely `dds_device_server` unless otherwise noted.
 
-### Camera's server
+# Usage
 
-Connected directly to the camera, the server represents it on the network publishes its availability and shares it's data.
+To enable DDS devices in *librealsense*, make sure `BUILD_WITH_DDS` is turned on in CMake. Make sure to check out the [`rs-dds-server` tool](../../tools/dds/dds-server) to turn any camera into a DDS device.
 
-The main objects that can be used in this application type
-* **dds_device_broadcaster** publishes the *device-info* topic, informing subscribers on cameras availability.
-* **dds_device_server** represents a camera at DDS level, manages streams, publishes images and receives controls from the user.
-* **dds_stream_server** distributes stream images into a dedicated topic
+The client for DDS devices is the librealsense Viewer: when built with `BUILD_WITH_DDS`, the Viewer will discover any devices available and should be able to stream from them, control them, and generally treat them like any other device.
```

Comment:
The client can be any user of context, like tools and examples other than the viewer

Reply:
Now `The client for DDS devices is librealsense and anything built with it, notably the Viewer: ...`